src/cli/argv_parser.js

```diff
@@ -79,6 +79,11 @@ export default class ArgvParser {
         ArgvParser.mergeJson('--format-options'),
         {}
       )
+      .option(
+        '--id-generator <OPTION>',
+        'how to generate ids used in envelopes (options: "uuid", "incrementing". default: "uuid")',
+        'uuid'
+      )
```

Comment:
TODO update to `--predictable-ids` following gherkin and add note that it is ignored in parallel mode

Reply:
Done